src/utils/navigation.ts

```diff
@@ -42,3 +42,18 @@ export const redirectToSavedUrl = (
  * @returns {string} sign in path
  */
 export const buildSignInPath = ({ host }: { host: string }) => `${host}/signin`;
+/**
+ * @param  {string} host target host
+ * @param  {string} itemId id of the item
+ * @param  {boolean} chatOpen whether to have the chat open
+ * @returns {string} lint to item with chat open
+ */
+export const buildItemLink = ({
```

Comment:
I know it's builder, but since perform has a different link for items, maybe it should be contained in the name of the function? Also you can use `qs` to handle query strings!